src/GeneralStateTestsFiller/stCreateTest/CREATE_EOF1Filler.yml

```diff
@@ -41,33 +41,239 @@ CREATE_EOF1:
       #- ':yul { mstore8(0, 0xef) return(0, 32) }'
       - ':label deploying_0xef00..00 :raw 0x60ef60005360206000f3'
 
-      # Valid EOF1 initcode and code combinations (both with and withoug data section)
+      # Valid EOF1 initcode and code combinations (both with and without data section)
 
       # Legacy initcode => EOF code
       # deployed code without data section
       # deployed code section: 'PUSH1 0 STOP'
       - ':label deploying_valid_eof1 :yul { mstore(0, 0xef00010100030060000000000000000000000000000000000000000000000000) return(0, 10) }'
-      # deployed code with data section
+      # deployed code section: 'PUSH1 0 PUSH1 0 RETURN'
+      - ':label deploying_valid_eof1_return :yul { mstore(0, 0xef00010100050060006000f30000000000000000000000000000000000000000) return(0, 12) }'
+      # deployed code section: 'PUSH1 01 PUSH1 00 SSTORE REVERT'
```

Comment:
Maybe add another `PUSH1 00 PUSH1 00` before `REVERT` for this to stay valid with EIP-5450 (`REVERT` requires 2 stack values)

Reply:
Fixed